documentation/sphinx/source/release-notes/release-notes-630.rst

```diff
@@ -90,7 +112,7 @@ Status
 * Removed fields ``worst_version_lag_storage_server`` and ``limiting_version_lag_storage_server`` from the ``cluster.qos`` section. The ``worst_data_lag_storage_server`` and ``limiting_data_lag_storage_server`` objects can be used instead. `(PR #3196) <https://github.com/apple/foundationdb/pull/3196>`_
 * If a process is unable to flush trace logs to disk, the problem will now be reported via the output of ``status`` command inside ``fdbcli``. `(PR #2605) <https://github.com/apple/foundationdb/pull/2605>`_ `(PR #2820) <https://github.com/apple/foundationdb/pull/2820>`_
 * When a configuration key is changed, it will always be included in ``status json`` output, even the value is reverted back to the default value. [6.3.5] `(PR #3610) <https://github.com/apple/foundationdb/pull/3610>`_
-* Added transactions.rejected_for_queued_too_long for bookkeeping the number of transactions rejected by commit proxy because its queuing time exceeds MVCC window. `(PR #4353) <https://github.com/apple/foundationdb/pull/4353>`_
+* Added transactions.rejected_for_queued_too_long for bookkeeping the number of transactions rejected by commit proxy because its queuing time exceeds MVCC window.[6.3.11] `(PR #4353) <https://github.com/apple/foundationdb/pull/4353>`_
```

Comment:
The convention for release notes is that we include all release notes for all patch version prior to an officially sanctioned release in one group. That means that if 6.3.11 is your first officially published release, every patch from 6.3.0 to 6.3.11 would be grouped together and you would identify patch changes like has been done here. 

After a release is officially published, all new changes are placed in their own sections and not mingled with the rest.

This PR seems to be combining the two approaches, which we should try to avoid. I think we haven't officially released, in which case all of them should be grouped under a single heading. Or you could move this change up to 6.3.11 if you think it does deserve its own heading.

Reply:
Yeah, I noticed that. Because the PR already has a release note at this place, so I didn't move it. 

We probably need to re-arrange the release notes for the official release.